src/components/TestSiteList.js

```diff
@@ -255,4 +179,6 @@ class TestSiteList extends React.Component {
     }
 }
 
-export default TestSiteList;
+export default GoogleApiWrapper({
+    apiKey: 'AIzaSyCj5wGAsi1ppD8qf6Yi-e6fMChdck7BMVg'
```

Comment:
@zboldyga is this something we want out there?

Reply:
@patrickpierson yep this key is OK, it's restricted to only be usable from our domains